host/xtest/xtest_main.c

```diff
@@ -150,6 +154,12 @@ int main(int argc, char *argv[])
 
 	opterr = 0;
 
+	if (argc > 1 && !strcmp(argv[1], "sha-perf")){
```

Comment:
It's not a big issue, if you want two ways of doing the same thing it's fine with me. :-)